index.bs

```diff
@@ -3871,30 +3896,36 @@ TransformStream(<var>transformer</var> = {}, writableStrategy = undefined, { <va
   how the constructed stream instance behaves:
 
   <ul>
-    <li> <code>start(controller)</code> is called immediately, and is typically used to enqueue prefix data that appears
-      in the output but doesn't depend on the input. If this process is asynchronous, it can return a promise to signal
-      success or failure.
-    <li> <code>transform(chunk, controller)</code> is called when a new <a>chunk</a> of input data is ready to be
-      transformed. It can return a promise to signal success or failure of the transformation. The results of the
-      transformation can be appended to the output using the {{ReadableStreamDefaultController/enqueue(chunk)}} method.
-      This permits a single input chunk to result in zero or multiple output chunks. The stream implementation
-      guarantees that <code>transform</code> will be called only after previous transformations have succeeded, and
-      never before <code>start</code> has completed or after <code>flush</code> is called. When no
-      <code>transform</code> method is supplied the identity transform is used, which copies chunks unchanged from the
-      input to the output.
-    <li> <code>flush(controller)</code> is called after all input chunks have been transformed. It is typically used to
-      append a suffix to the output data. If this process is asynchronous, it can return a promise
-      to signal success or failure. On success, the {{readable}} side will be closed.
+    <li><p><code>start(controller)</code> is called immediately, and is typically used to enqueue prefix <a>chunks</a>
+      that appear in the <a>readable side</a> but doesn't depend on any writes to the <a>writable side</a>. If this
```

Comment:
"... **don't** depend on ..."

"appear" is awkward here. Maybe "will be read from"?

Reply:
Done.